src/components/Applications.jsx

```diff
@@ -47,7 +47,9 @@ const getApplicationsList = memoize(data => {
         !flag(`home_hidden_apps.${app.slug.toLowerCase()}`) // can be set in the context with `home_hidden_apps: - drive - banks`for example
     )
     const dedupapps = uniqBy(apps, 'slug')
-    const array = dedupapps.map(app => <AppTile key={app.id} app={app} />)
+    const array = dedupapps
```

Comment:
suite à la description "This filter should be temporary as we expect to hide the cozy-app at a
higher level on cozy-client collections" on devrait peut-être mettre aussi un commentaire dans le code

Reply:
[Corrigé](https://github.com/cozy/cozy-home/compare/89ef9e2cedf2f6af2df9f586f8fe92c136ae4fee..0456cb99f327d37083d3998624c914782ece3f2a)